### DISCUSSION TITLE
Are Kotlin contributions accepted?

### DISCUSSION THREAD

**Discussion by AbdelHajou** [DependencyTrack/dependency-track#1416] [Q&A]
Hello,

I am thinking about becoming a long-term contributor to this project. I was wondering whether this project accepts Kotlin contributions. Kotlin is fully interoperable with Java, so we can call Kotlin code from the existing Java code and vice versa.

Please let me know if there are any plans for adding Kotlin modules and how I could contribute to this. Thanks!

**Comment by stevespringett** [comment]
The project will be modularizing a bit. Currently, its a monolith and we're moving to more of a domain service model (not microservice). With modularization, there may be an opportunity to introduce other languages. One module for example, may be written in Java or Go, we haven't decided yet.
